gateway/version_info.py

```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+
+from datetime import datetime
+
+@dataclass(frozen=True)
+class VersionInfo:
+    version: int
+    author: str
+    date: datetime
```

Comment:
I'd opt for `datetime` or `timestamp`. I find it confusing when only `date` is used for both date and time.

Reply:
Looks as if you're using the data straight from ROCFL, so maybe this would be some work to change...

Reply:
Yes. We can't (readily) translate from the `rocfl` log back into discrete agent/contact, and we only care about the `version` number at the moment.